Lookuptable/README.md

```diff
@@ -0,0 +1,57 @@
+## Commcare lookuptable Test Script
+
+Lookup table tests ensure that the important features are working and there are no showstoppers in the build deployed to environments.\
```

Comment:
Let's update the description here to:

`These tests ensure that the lookup table features work as expected and that there are no regressions.`

Reply:
Let's update the description here to:

```suggestion
These tests ensure that the [lookup table ](https://confluence.dimagi.com/display/commcarepublic/Lookup+Tables)features work as expected and that there are no regressions.

```